doc/using-logging.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+ Configuring Logging
+=====================
+
+One common use of ZConfig is to configure the Python :mod:`logging`
+framework. ZConfig provides one simple convenience function to do this:
+
+.. autofunction:: ZConfig.configureLoggers
+
+
+This function takes the configuration text::
+
+    >>> from ZConfig import configureLoggers
+    >>> configureLoggers('''
```

Comment:
I can and will include (and expand on) the example by itself in the section about the configuration format. 

I'd prefer not go to the trouble of adding a dependency on manuel, setting up a suite for it and testing these few lines---it seems like logging configuration and output is going to be one of those things that causes issues under doctests in the long run. If you feel strongly, though, I can look into it.